app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] Kindly make sure the `users_controller` has both `#index` and `#show` actions(It's okay for the actions to be empty for now) but make sure to add them for the project requirement.

Reply:
Done, I fixed it.